README.md

```diff
@@ -13,39 +13,45 @@ You can use AgentAPI:
 
 ## Quickstart
 
-1. Install `agentapi` by downloading the latest release binary from the [releases page](https://github.com/coder/agentapi/releases).
+1. Install `agentapi`:
 
-1. Verify the installation:
+   ```bash
+   sh -c 'curl -fsSL https://github.com/coder/agentapi/releases/latest/download/agentapi-$(uname -s | tr "[:upper:]" "[:lower:]")-$(uname -m | sed "s/x86_64/amd64/;s/aarch64/arm64/") -o agentapi' && chmod +x agentapi
```

Comment:
Why wrap this in `sh -c`?

Also, consider breaking out the two sub-commands for legibility, e.g.

```
OS=$(uname -s | tr "[:upper:]" "[:lower:]");
ARCH=$(uname -m | sed "s/x86_64/amd64/;s/aarch64/arm64/");
curl -fsSL "https://github.com/coder/agentapi/releases/latest/download/agentapi-${OS}-${ARCH}" -o agentapi && chmod +x agentapi
```